structs/ctl/types.go

```diff
@@ -93,8 +99,18 @@ type JoinInfo struct {
 	Address     string `json:"address"`
 }
 
-type AddMemberInfo struct {
-	CommunityID string `json:"community_id"`
-	Address     string `json:"address"`
-	NodeID      string `json:"node_id"`
+// Sent by server back to client
```

Comment:
somewhere can you document the steps of the first time handshake /  creating a cert / signing it / authenticating with it between the server & client?

Reply:
Yeah definitely